server/ctrlbase/ctrl.go

```diff
@@ -47,6 +47,7 @@ func statusToBlock(code int) string {
 type Controller struct {
 	DB          *db.DB
 	MusicPath   string
+	CachePath   string
```

Comment:
i think this can be moved to the Controller in `ctrlsubsonic` instead, since it's only used there (instead of here where both the admin page and subsonic use it)

Reply:
How do I do it properly? AFAICT, at the moment `ctrlsubsonic` gets spawned by `server` with all options copied from `ctrlbase`.

Reply:
oops you'd need to pass an arg to SetupSubsonic. nevermind then. i need to refactor server.go anyway so i'll do it then